packages/itwin/grouping-mapping-widget/src/widget/components/GroupingMappingContext.tsx

```diff
@@ -44,13 +46,15 @@ export interface GroupingMappingContextProps {
    * Custom UI to add and update groups or provide additional group context capabilities.
    */
   customUIs?: GroupingMappingCustomUI[];
+  iModelConnection?: IModelConnection;
   children?: React.ReactNode;
 }
 
 const authorizationClientGetAccessToken = async () =>
   (await IModelApp.authorizationClient?.getAccessToken()) ?? "";
 
 export const GroupingMappingContext = (props: GroupingMappingContextProps) => {
+  const iModelConnectionUIF = useActiveIModelConnection();
```

Comment:
What does `UIF` stand for?

Reply:
It stood for UI Framework. I now changed it to `activeIModelConnection`